OracleWebCenterSites/README.md

```diff
@@ -1,461 +1,45 @@
-Before you build an Oracle WebCenter Sites image, download the Oracle Fusion Middleware infrastructure and Oracle Database images from the [Docker Store.](https://store.docker.com/)
+Before you build, select the version and distribution for which you want to build an image, then download the required packages (see `.download` files) and place them in the folder of your distribution version of choice. Then, from the `dockerfiles` folder, run the `buildDockerImage.sh` script.
 
-If you download Oracle Fusion Middleware infrastructure from Docker Store, then retag using below command:
-```
-   $ docker tag store/oracle/fmw-infrastructure:12.2.1.3 oracle/fmw-infrastructure:12.2.1.3
-```
-If you download Oracle Database from Docker Store, then retag using below command:
-```
-   $ docker tag store/oracle/database-enterprise:12.2.0.1 database/enterprise:12.2.0.1
-```
-##### 9. How do I build an Oracle Database 12.2.1.x base image?
-If you want to build your own Oracle Database image, use the Docker files and scripts in the [Oracle Database](../OracleDatabase) GitHub repository.
+		-bash-4.2$ sh buildDockerImage.sh -h
+		
+		Usage: buildDockerImage.sh -v [version] [-s] [-c]
+		Builds a Docker Image for Oracle WebCenter Sites.
+		
+		Parameters:
+				-v: version to build. Required.
+						Choose one of: 12.2.1.3  12.2.1.4
+				-c: enables Docker image layer cache during build
+				-s: skips the MD5 check of packages
+		
+		LICENSE CDDL 1.0 + GPL 2.0
+		
+		Copyright (c) 2017 Oracle and/or its affiliates. All rights reserved.
+		
+		Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
 
-##### 10. How do I build an Oracle Fusion Middleware Infrastructure 12.2.1.x base image?
-If you want to build your own Oracle Fusion Middleware Infrastructure image, use the Docker files and scripts in the [Oracle FMW Infrastructure](../OracleFMWInfrastructure) GitHub repository.
+**IMPORTANT**
 
-##### 11. How to fix yum.oracle.com connectivity error?
-The errors mean that the host is not able to connect to external registries for update. To access external registries and build a Docker image, set up environment variables for proxy server as below:
-```
-   export http_proxy=http://www-yourcompany.com:80 
-   export https_proxy=http://www-yourcompany.com:80 
-   export HTTP_PROXY=http://www-yourcompany.com:80 
-   export HTTPS_PROXY=http://www-yourcompany.com:80 
-   export NO_PROXY=localhost,.yourcompany.com 
-```
-##### 12. How to fix error "Please specify script.work.dir to use an alternate location” while running Admin Container?
-Make sure you have granted the right permission to 'oracle' user as described in section [Mounting a Host Directory as a Data Volume](#b-mounting-host-directory-as-a-data-volume-1).
+   For detail Oracle WebCenter Sites 12.2.1.3.0 set up, refer [this](dockerfiles/12.2.1.3).
+        
+   For detail Oracle WebCenter Sites 12.2.1.4.0 set up, refer [this](dockerfiles/12.2.1.4).
```

Comment:
This should also be a bullet point and as its newer, should be listed first, i.e.
```
* [Oracle WebCenter Sites 12.2.1.4.0](dockerfiles/12.2.1.4)
```